content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,74 @@
+
+## Redefining Dual Stack Support
+
+Much of the feedback provided by the community for the original RFC was to update Envoy to better support dual-stack use cases
+internally instead of supporting this within Istio. This has led us to a [new design](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing)
+where we have taken lessons learned as well as feedback and have applied them to fit a simplified design.
+
+## Support for Dual Stack in Istio 1.17
+
+We have worked with the Envoy community to address numerous concerns which is a reason why dual-stack enablement has
+taken us a while to implement. We have implemented [matched IP Family for outbound listener](https://github.com/envoyproxy/envoy/issues/16804)
+and [supported multiple addresses per listener](https://github.com/envoyproxy/envoy/issues/11184). Alex Xu has also
+been working fervently to get long outstanding issues resolved, with the ability for Envoy to have a
+[smarter way to pick endpoints for dual-stack](https://github.com/envoyproxy/envoy/issues/21640). Some of these improvements
+to Envoy, such as the ability to [enable socket options on multiple addresses](https://github.com/envoyproxy/envoy/pull/23496),
+have landed in the Istio 1.17 release (e.g. [extra source addresses on inbound clusters](https://github.com/istio/istio/pull/41618)).
+
+The Envoy API changes made by the team can be found at their site at [Listener addresses](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto.html?highlight=additional_addresses)
+and [bind config](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-bindconfig).
+Making sure we can have proper support at both the downstream and upstream connection for Envoy is important for realizing
+dual-stack support.
+
+In total the team has submitted over a dozen PRs to Envoy and are working on at least a half dozen more to make Envoy adoption of
+dual stack easier for Istio.
+
+Meanwhile, on the Istio side you can track the progress in [Issue #40394](https://github.com/istio/istio/issues/40394).
+Progress has slowed down a bit lately as we continue working with Envoy on various issues, however, we are happy to
+announce experimental support for dual stack in Istio 1.17!
+
+### Get involved
+
+Plenty of work remains and you are welcome to help us with the remaining tasks needed for dual stack support to get to Alpha [here](https://github.com/istio/enhancements/pull/141).
+
+For instance, Iris Ding (Intel) and Li Chun (Intel) are already working with the community for getting redirection of
+network traffic for ambient and we are hoping to have ambient support dual stack for its upcoming alpha release in
```

Comment:
could you share some future thoughts on ambient for this besides traffic redirection? Does this change ztunnel configuration as well?

Reply:
This is an area I'm not too familiar with. @zhlsunshine and @irisdingbj do you have any more thoughts?

Reply:
I think @irisdingbj can provide more info about this.